src/main/java/com/hibernate/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.hibernate.service.impl;
+
+import com.hibernate.dao.OrderDao;
+import com.hibernate.lib.Inject;
+import com.hibernate.lib.Service;
+import com.hibernate.model.Order;
+import com.hibernate.model.ShoppingCart;
+import com.hibernate.model.User;
+import com.hibernate.service.OrderService;
+import com.hibernate.service.ShoppingCartService;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
```

Comment:
is it ok to use one service inside another one? may be it would be better to use correspondent Dao here